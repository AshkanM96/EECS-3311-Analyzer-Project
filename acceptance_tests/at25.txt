add
type_check
reset
reset
type_check
add
boolean_constant(false)
type_check
evaluate
integer_constant(1)
type_check
evaluate
reset