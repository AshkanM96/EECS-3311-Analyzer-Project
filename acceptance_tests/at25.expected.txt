  Expression currently specified: ?
  Report: Expression is initialized.
->add
  Expression currently specified: (? + nil)
  Report: OK.
->type_check
  Expression currently specified: (? + nil)
  Report: Error (Expression is not yet fully specified).
->reset
  Expression currently specified: ?
  Report: OK.
->reset
  Expression currently specified: ?
  Report: Error (Initial expression cannot be reset).
->type_check
  Expression currently specified: ?
  Report: Error (Expression is not yet fully specified).
->add
  Expression currently specified: (? + nil)
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (False + ?)
  Report: OK.
->type_check
  Expression currently specified: (False + ?)
  Report: Error (Expression is not yet fully specified).
->evaluate
  Expression currently specified: (False + ?)
  Report: Error (Expression is not yet fully specified).
->integer_constant(1)
  Expression currently specified: (False + 1)
  Report: OK.
->type_check
  Expression currently specified: (False + 1)
  Report: (False + 1) is not type-correct.
->evaluate
  Expression currently specified: (False + 1)
  Report: Error (Expression is not type-correct).
->reset
  Expression currently specified: ?
  Report: OK.
